app/scripts/directives/necesidad/visualizar_necesidad.js

```diff
@@ -115,12 +116,20 @@ angular.module('contractualClienteApp')
                                 if (solicitante) {
                                     self.dependencia_solicitante = response_dependencia.data[0];
                                     //TODO: Arreglar llamado a metasRequest
-                                    metasRequest.get('plan_adquisiciones/' + $scope.necesidad.Necesidad.Vigencia +
-                                        "/" + response_dependencia.data[0].Id).then(function (response) {
-
+                                    metasRequest.get('Registro_plan_adquisiciones-Metas_Asociadas?query=RegistroPlanAdquisicionesId__PlanAdquisicionesId__Vigencia%3A' + $scope.necesidad.Necesidad.Vigencia +
+                                        "%2CRegistroPlanAdquisicionesId__ResponsableId%3A" + response_dependencia.data[0].Id + "&limit=-1").then(function (response) {
```

Comment:
Mucha responsabilidad en una misma línea...
```suggestion
                                    const queryMetas = 'Registro_plan_adquisiciones-Metas_Asociadas?query=RegistroPlanAdquisicionesId__PlanAdquisicionesId__Vigencia%3A' + $scope.necesidad.Necesidad.Vigencia +
                                    "%2CRegistroPlanAdquisicionesId__ResponsableId%3A" + response_dependencia.data[0].Id + "&limit=-1";
                                    metasRequest.get(queryMetas).then(function (response) {
```